client/checkout/classic/index.js

```diff
@@ -79,12 +81,24 @@ jQuery( function ( $ ) {
 		type: 'sofort' /* eslint-disable camelcase */,
 	};
 
+	let isUPEComplete = false;
+	const upeElement = paymentIntent
```

Comment:
This is more a thought/observation, but is using the existence of the PI enough of a flag for UPE or should we use something explicit to signify we're using UPE? I think it's something we can address in the future if we ever need to, just wanted to put it out there.

Reply:
Good point: I just added [this config value](https://github.com/Automattic/woocommerce-payments/blob/1e748f957dd221d66cf40aec2b83a9d7ea310fee/client/checkout/classic/index.js#L22), which gets set [here](https://github.com/Automattic/woocommerce-payments/blob/1e748f957dd221d66cf40aec2b83a9d7ea310fee/includes/class-wc-payment-gateway-wcpay.php#L505) in the main payment gateway.